src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,59 @@
+.nav {
+  &--margin {
+    margin-right: 50px;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    position: relative;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    font-size: 12px;
+    align-items: center;
+    height: 60px;
+    margin-right: 20px;
+    position: relative;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    color: $textColor;
+    height: 100%;
+  }
+
+  &__link.is-active {
+    color: $activeColor;
+  }
+
+  &__link:hover {
+    color: $activeColor;
+    transition: 0.2ms;
+  }
+
+  &__link.is-active::after {
```

Comment:
Nesting

Reply:
Also don't increase specificity if no needed. Move `.is-active` class up from this block(use only `.is-active` without `&__link`)